datavec/datavec-api/src/test/java/org/datavec/api/transform/ops/AggregableMultiOpArchTest.java

```diff
@@ -0,0 +1,31 @@
+package org.datavec.api.transform.ops;
+
+import com.tngtech.archunit.core.importer.ImportOption;
+import com.tngtech.archunit.junit.AnalyzeClasses;
+import com.tngtech.archunit.junit.ArchTest;
+import com.tngtech.archunit.junit.ArchUnitRunner;
+import com.tngtech.archunit.lang.ArchRule;
+import org.junit.runner.RunWith;
+import org.nd4j.common.tests.BaseND4JTest;
+
+import java.io.Serializable;
+
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.classes;
+
+/**
+ * Created by dariuszzbyrad on 7/31/2020.
+ */
+@RunWith(ArchUnitRunner.class)
+@AnalyzeClasses(packages = "org.datavec.api.transform.ops", importOptions = {ImportOption.DoNotIncludeTests.class})
+public class AggregableMultiOpArchTest extends BaseND4JTest {
+
+    @ArchTest
+    public static final ArchRule ALL_AGGREGATE_OPS_MUST_BE_SERIALIZABLE = classes()
```

Comment:
Potential error looks like:
_Class <org.datavec.api.transform.ops.DispatchOpTest> does not implement java.io.Serializable_